content/docs/sidebar.json

```diff
@@ -154,6 +154,10 @@
           "share-a-dvc-cache"
         ]
       },
+      {
+        "label": "Plots Configuration and Visualization",
+        "slug": "plots"
```

Comment:
Name's a bit long. It'd be best if it fits in one line in the nav.

<img width="399" alt="image" src="https://user-images.githubusercontent.com/1477535/182013944-eec20bfd-2842-4fff-b8ec-cb69cadb9dcd.png">

Also that the slug marches the title if possible (better for SEO AFAIK).

Reply:
Agreed. Do you think `Plots` is enough? What do you suggest?

Reply:
Maybe "Data Visualization" for now?

Or something else that will let us include params and metrics (in general) in the future too, per https://github.com/iterative/dvc.org/issues/2572 but we can change it later.

Reply:
> Maybe "Data Visualization" for now?

I like to at least use the word "Plots" or it feels like needless obfuscation. Do you think so, or is "Data Visualization" more accurate in some way?

> Or something else that will let us include params and metrics (in general) in the future too, per https://github.com/iterative/dvc.org/issues/2572 but we can change it later.

I feel plots may need its own guide since it's a fairly complex subsystem of DVC on its own, but yeah, I guess it's premature to decide either way right now.

Reply:
Maybe this for now
```suggestion
        "slug": "visualizing-plots"
```
Up to you, I just think just "Plots" is too generic. But we expect guide traffic to come from links in other docs (rather than by browsing the nav) so the name may not be that important.

Reply:
`Visualizing plots` works and fits the style of the rest of the other guides, so I'll go with that.

This is mostly a style choice, but I prefer to keep it simple because IMHO it's easier to navigate if the sidebar looks like:

* Experiments
* Pipelines
* Plots

It takes longer for me to find the right section when we add in these descriptive words like:

* Experiments management
* Data pipelines
* Visualizing plots

Reply:
Goes back to how exactly we want to organize the user guide. I don't think we can maintain detailed guides for every single feature so specific titles may be more appropriate. Up to you